lang/rust/rust.talon

```diff
@@ -0,0 +1,63 @@
+mode: command
+and mode: user.rust
+mode: command
+and mode: user.auto_lang
+and code.language: rust
+-
+tag(): user.code_imperative
```

Comment:
Did you want to implement the `code_block` action too, for the `block` command?

Reply:
> Did you want to implement the `code_block` action too, for the `block` command?

Yeah, thanks for pointing this out.

And about formatters - can you give an example of what you mean?